prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -347,13 +347,16 @@ WORKFLOW= # Make sure we start with an empty workflow
 # ---------------------------------------------------------------------------------------------------------------------
 # Input workflow
 INPUT_DETECTOR_LIST=$WORKFLOW_DETECTORS
-if [[ ! -z ${WORKFLOW_DETECTORS_USE_GLOBAL_READER} ]]; then
-  for i in ${WORKFLOW_DETECTORS_USE_GLOBAL_READER//,/ }; do
+if [[ ! -z ${WORKFLOW_DETECTORS_USE_GLOBAL_READER_TRACKS} ]] || [[ ! -z ${WORKFLOW_DETECTORS_USE_GLOBAL_READER_CLUSTERS} ]]; then
```

Comment:
In case an env variable is not necessarily defined you need to put this extension `${WORKFLOW_DETECTORS_USE_GLOBAL_READER_TRACKS:-}`. Otherwise you get the error as in the fullCI `/sw/slc8_x86-64/O2/12680-slc8_x86-64-local1/prodtests/full-system-test/dpl-workflow.sh: line 350: WORKFLOW_DETECTORS_USE_GLOBAL_READER_TRACKS: unbound variable`